setup.cfg

```diff
@@ -13,13 +13,13 @@ classifiers =
 
 [options]
 packages =
-	edgepi_rpc_client,
-	edgepi_rpc_client.adc,
-	edgepi_rpc_client.dac,
-	edgepi_rpc_client.din,
-	edgepi_rpc_client.dout,
-	edgepi_rpc_client.led,
-	edgepi_rpc_client.relay,
+	edgepi_rpc_client
```

Comment:
I don't think you would need to list all sub modules like that. what happens when you just have edgepi_rpc_client ?

Reply:
I tried just having `edgepi_rpc_client` but that caused import errors when installing from the server. I could do it this way or alternatively have it the way I had previously, with `packages: find:`, which located all modules in the directory.

Reply:
keep it as is